packages/app/package.json

```diff
@@ -17,8 +17,8 @@
     "test:watch": "jest --watch --env=jsdom",
     "test:integrations": "jest --config integration-tests/jest.config.json --maxWorkers=2",
     "lint": "npm run lint:app && npm run lint:embed",
-    "lint:app": "cross-env LINT=1 eslint src/app",
-    "lint:embed": "cross-env LINT=1 eslint src/embed",
+    "lint:app": "cross-env LINT=1 eslint --ext .js,.ts,.tsx src/app",
```

Comment:
It's weird that we explicitly have to set the extensions.
I would assume ESLint will automatically run on `.js` files and with the `@typescript-eslint/parser` that it would automatically run on `.ts(x)` files too, but happy if it works like this 🙂

Reply:
Not that weird, from [command line docs](https://eslint.org/docs/user-guide/command-line-interface):
```
--ext [String]                 Specify JavaScript file extensions - default: .js
```

Reply:
Maybe not that weird then indeed.